utility/src/main/java/org/oppia/util/networking/NetworkConnectionUtil.kt

```diff
@@ -24,11 +24,14 @@ class NetworkConnectionUtil @Inject constructor(private val context: Context) {
     testConnectionStatus?.let {
       return it
     }
-    val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+    val connectivityManager = context.getSystemService(
+      Context.CONNECTIVITY_SERVICE) as ConnectivityManager
```

Comment:
Eh, probably should have ``)`` on its own line, too, though we should break at ``=`` first.